components/support/jwcrypto/src/ec.rs

```diff
@@ -2,33 +2,39 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+//! Implements Elliptic-Curve Diffie-Hellman for JWE.
```

Comment:
Since you're here, let's also link out to the relevant RFC sections, so it's clear exactly what we're trying to do. In this case it's [Section 4.6](https://tools.ietf.org/html/rfc7518#section-4.6) and specifically the "direct key agreement" mode.